pysteps/visualization/motionfields.py

```diff
@@ -26,11 +25,10 @@ def quiver(
     map=None,

```

Comment:
Using **map** directly as keyword masked the build-in `map` python function. Although the code works (since the reference to map is redefined) maybe we can move map to kwargs and assign its value to a different variable name `plot_map=kwargs.pop("map")`. By doing this we don't need to change the interface and we avoid the shadowing of the python `map` function. What do you think?

Reply:
It's an excellent point, but unfortunately this is true not only for the quiver() function, but also for all our plotting methods.

We could implement your workaround for most cases,  but I feel that this would somewhat complicate the code. 
Alternatively we could explicitly rename the `map` argument to `plot_map`, but this will change the interface.

I understand that the current situation is not ideal, but do you see any major problem in leaving it as it is? Our local `map` argument shadows the built-in `map` method, which is not used in there. Do you think that something can go wrong by doing it?

Reply:
> Do you think that something can go wrong by doing it?

I don't think so. We can leave it as it is. But in the long term, maybe we can change the interface slowly, first adding a warning that the argument name will change, and a few releases down the road, change the implementation. 
Either way, this is not an important issue to solve right now, but we should keep it in mind for future releases :)

Reply:
Right, good idea. In cc505e2  I introduced a FutureWarning and internally renamed `map` to `plot_map`, do you mind having a quick a look if this looks OK to you?

Reply:
Great! I also added some minor changes to the `plot_geography` interface in commits 9714c3f and 9d0ed8a.